roles/tackle/defaults/main.yml

```diff
@@ -161,6 +161,24 @@ analyzer_container_limits_memory: "4Gi"
 analyzer_container_requests_cpu: "1"
 analyzer_container_requests_memory: "4Gi"
 
+provider_generic_component_name: "addon"
```

Comment:
Should this ^ be "addon"?

Reply:
I don't know. It's going to end up being used in the provider_java_service_name to generate something like {konveyor,mta}-generic-addon. 

It looks like it's going to get applied as a label, which I am guessing is at least informational to the user.

If it makes sense we can change it to `provider`.

Reply:
_extension_ would be more accurate.